apps/codeforafrica/src/lib/index.js

```diff
@@ -277,6 +277,45 @@ export const team = [
   },
 ];
 
+const guidingPrinciples = [
+  {
+    icon: "/icons/About icon.png",
```

Comment:
```suggestion
    icon: {
      src: "/icons/About icon.png",
    },
```
I think we use this format everywhere we need images; `image`, `thumbnail`, `icon`, etc.